exercises/isbn-verifier/description.md

```diff
@@ -2,7 +2,7 @@ Check if a given ISBN-10 is valid.
 
 ## Functionality
 
-Given an unkown string the program should check if the provided string is a valid ISBN-10.
```

Comment:
http://cf.geekdo-images.com/images/pic1453075.jpg

Reply:
In every exchange with you, I learn something.